pkgs/top-level/all-packages.nix

```diff
@@ -17848,6 +17848,19 @@ in
     ];
   };
 
+  # Needs a `date` and a `sha256` argument.
+  # Sometimes the version doesn't align with `linux_testing`. In that case specify version by hand.
+  linux_next = args: callPackage ../os-specific/linux/kernel/linux-next.nix ({
+    sha256 = "eeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeee";
```

Comment:
you could use `lib.fakeSha25`

Reply:
This sounds interesting, didn't know about it yet! I still am unsure, if I want the download to happen with a wrong hash by default or simply having a missing argument and hence forcing the user to supply one.

But probably this is the best way. Will update to use this!